### DISCUSSION TITLE
How much time dooes the http scaler wait before scaling the app to 0

### DISCUSSION THREAD

**Discussion by Kaitou786** [kedacore/http-add-on#550] [Q&A]
Hi, I have noticed for scaling to 0 I have noticed that it waits around 10 mins before scaling it to 0, is that correct, and can we configure or change that behavior somehow?

**Comment by someshkoli** [comment]
If this is made configurable, it'll be very handy. Checking if somehow I can build this

**Reply by tomkerkhove** [reply]
KEDA Core has this as part of the ScaledObject so it should be feasible to do this.

Looks like you've opened https://github.com/kedacore/http-add-on/issues/562 to track this.

**Comment by linkvt** [comment]
Closing as outdated — this predates several releases and either got resolved since or never reached a conclusion. See current docs (https://keda.sh/http-add-on/0.15/) or feel free to open a new discussion if still relevant.
